openvdb/points/AttributeArray.h

```diff
@@ -503,6 +640,36 @@ class TypedAttributeArray: public AttributeArray
         return hasConstantStride() ? mSize * mStrideOrTotalSize : mStrideOrTotalSize;
     }
 
+#if OPENVDB_ABI_VERSION_NUMBER >= 6
+    /// Return the name of the value type of a single element in this array (e.g., "float" or "vec3d").
+    Name valueType() const override { return typeNameAsString<ValueType>(); }
+
+    /// Return the name of the codec used by this array (e.g., "trnc" or "fxpt").
+    Name codecType() const override { return Codec::name(); }
+
+    /// Return the size in bytes of the value type of a single element in this array.
+    Index valueTypeSize() const override { return sizeof(ValueType); }
+
+    /// Return the size in bytes of the storage type of a single element of this array.
+    /// @note If the Codec is a NullCodec, valueSize() == storageSize()
+    Index storageTypeSize() const override { return sizeof(StorageType); }
+
+    /// Return @c true if the value type is floating point
+    bool valueTypeIsFloatingPoint() const override;
+
+    /// Return @c true if the value type is scalar (not vector, quaternion or matrix)
```

Comment:
Think this comment is the wrong way round?

Reply:
Thanks, just resolved that.